packages/trader/src/Stores/Modules/SmartChart/Helpers/__tests__/barriers.ts

```diff
@@ -34,17 +34,17 @@ describe('Barriers', () => {
             const key_to_remove = BARRIERS_KEYS.TAKE_PROFIT;
             Barriers.removeBarrier(barriers, key_to_remove);
             expect(barriers.find(barrier => barrier.key === key_to_remove)).toBeUndefined();
-            expect(barriers.length).toEqual(2);
+            expect(barriers).toHaveLength(2);
```

Comment:
@yauheni-deriv this change is not related to your PR. please update your local master, switch to this branch, and run:
`git checkout master packages/trader/src/Stores/Modules/SmartChart/Helpers/__tests__/barriers.ts`
commit & push. :)

Reply:
well, at least I've tried.
these changes had been done automatically by eslint. 
as. I see "eslint-plugin-jest" is a part of  "eslint-config-deriv"
So I can’t change it

Reply:
@yauheni-deriv  please do the following: 1. update your local master, 2. switch to this branch and run: git checkout master packages/trader/src/Stores/Modules/SmartChart/Helpers/__tests__/barriers.ts 3) commit with --no-verify & push